Lib/test/test_gc.py

```diff
@@ -766,6 +766,30 @@ def test_freeze(self):
         gc.unfreeze()
         self.assertEqual(gc.get_freeze_count(), 0)
 
+    def test_get_objects(self):
+        gc.collect()
+        l = []
+        l.append(l)
+        self.assertIn(l, gc.get_objects(generation=0))
+        self.assertNotIn(l, gc.get_objects(generation=1))
+        self.assertNotIn(l, gc.get_objects(generation=2))
+        gc.collect(generation=0)
+        self.assertNotIn(l, gc.get_objects(generation=0))
+        self.assertIn(l, gc.get_objects(generation=1))
+        self.assertNotIn(l, gc.get_objects(generation=2))
+        gc.collect(generation=1)
+        self.assertNotIn(l, gc.get_objects(generation=0))
+        self.assertNotIn(l, gc.get_objects(generation=1))
+        self.assertIn(l, gc.get_objects(generation=2))
+        gc.collect(generation=2)
+        self.assertNotIn(l, gc.get_objects(generation=0))
+        self.assertNotIn(l, gc.get_objects(generation=1))
+        self.assertIn(l, gc.get_objects(generation=2))
+
+        self.assertEqual(gc.collect(), 0)
```

Comment:
I don't think this part of the test is useful. We're testing `gc.get_objects`, not collection of cyclic references.